shell/src/main/java/alluxio/cli/fs/command/DistributedLoadCommand.java

```diff
@@ -233,10 +233,18 @@ public String getCommandName() {
 
   @Override
   public Options getOptions() {
-    return new Options().addOption(REPLICATION_OPTION).addOption(ACTIVE_JOB_COUNT_OPTION)
+    return new Options()
+        .addOption(REPLICATION_OPTION)
+        .addOption(ACTIVE_JOB_COUNT_OPTION)
         .addOption(INDEX_FILE)
         .addOption(HOSTS_OPTION)
         .addOption(HOST_FILE_OPTION)
+        .addOption(EXCLUDED_HOSTS_OPTION)
```

Comment:
Are these removed or missed from the beginning?

Reply:
They have been missing from the beginning https://github.com/Alluxio/alluxio/pull/13506/files#diff-60a4339f7e54f6750c208baea305dfcbfbb95d37645e64a13ca33090db580a35

Reply:
So these new options are added at that time, but not really worked?

Reply:
I think so.

Reply:
None of these new options have any kind of integration tests, either.